packages/auth/package.json

```diff
@@ -36,13 +36,13 @@
     "node-mocks-http": "^1.12.2",
     "objection": "^3.0.1",
     "objection-db-errors": "^1.1.2",
-    "open-payments": "workspace:*",
-    "openapi": "workspace:../openapi",
+    "@interledger/open-payments": "1.0.0-alpha.1",
```

Comment:
```suggestion
    "@interledger/open-payments": "1.0.0",
```
?

Reply:
We did `alpha` for now and will bump as soon as we are sure it all plays well together

Reply:
@sabineschaller I tested with the alpha versions previously in the PR, after publishing 1.0.0, we can use 1.0.0 for all of the packages, ie yes, that's correct @wilsonianb, I've just pushed the change